uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerFrom = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">To Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           show-button-bar="false"
+                           ng-model="modal.leaveRequest.toDate"
+                           is-open="modal.leaveRequest.showDatePickerTo"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerTo = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr/>
+          <div class="row form-group">
+            <div class="col-xs-4">Opening Balance</div>
+            <div class="col-xs-8">17.0</div>
+          </div>
+          <div class="row form-group text-primary pointer"
+               ng-click="modal.leaveRequest.isChangeExpanded = !modal.leaveRequest.isChangeExpanded">
+            <div class="col-xs-4">
+              <i class="fa chr-inline-icon"
+                 aria-hidden="true"
+                 ng-class="{ 'fa-chevron-right': !modal.leaveRequest.isChangeExpanded, ' fa-chevron-down': modal.leaveRequest.isChangeExpanded }"></i>
+              Change
+            </div>
+            <div class="col-xs-8">2.5</div>
+          </div>
+          <table class="table chr-table" ng-show="modal.leaveRequest.isChangeExpanded">
```

Comment:
Changed the name, but keeping `chr_table` as this table style would be resued for other pages also.